coins/src/utils/coins3/produce.ts

```diff
@@ -23,7 +23,7 @@ async function produceTopics(
 
   items.map((item) => {
     const { symbol, decimals } = item;
-    if (!symbol || !decimals) return;
+    if (!symbol || decimals == null) return;
```

Comment:
typeof decimals !== 'number' is safer? or it will skip decimals which are string?  hmm... maybe we can leave this as is
![image](https://github.com/user-attachments/assets/cf1e41cb-1320-4c5c-88d3-350fcdae3701)